tox.ini

```diff
@@ -12,9 +12,11 @@ install_command =
   pip install {opts} {packages}
 deps =
   mypy
-  stestr
+  pytest
+allowlist_externals = mkdir
 commands =
-  stestr run {posargs}
```

Comment:
Switching the executor for `stestr` (which apparently doesn't allow for exporting CircleCI-compatible test results, i.e. the ones in JUnit format) to `pytest`